core/src/impl/Kokkos_Core.cpp

```diff
@@ -29,7 +31,6 @@
 #include <algorithm>
 #include <cctype>
 #include <cstring>
-#include <iostream>
```

Comment:
Why did you change this?

Reply:
Because system headers come first, as per https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes. But I can revert this kind of changes if you prefer :wink:

Reply:
Ok, I'll revert that change :wink:

Reply:
Reverted.